src-built-in/components/windowTitleBar/src/components/center/TabRegion.jsx

```diff
@@ -18,7 +18,7 @@ let TAB_WIDTH = 300;
 const ICON_AREA = 29;
 const CLOSE_BUTTON_MARGIN = 22;
 const MINIMUM_TAB_SIZE = 100;
-
+import * as _throttle from "lodash.throttle";
```

Comment:
We should just move this up with the other imports so they are all together. Minor code nit.